.vscode/settings.json

```diff
@@ -1,8 +1,17 @@
 {
+  "prettier": {
```

Comment:
This prettier config shouldn't be here as it'll override any changes we make to the settings in our prettierrc files.

The `editor.defaultFormatter` setting below makes sense to leave here though